interchain/scripts/state.json

```
@@ -82,5 +82,28 @@
       "abstract:registry": "xion1j33edjnk9cqkuw83fvz3f646unxpkcjkru0znjhznr0dzvpzyw7q4yz2eu"
     },
     "version": "0.26.0"
+  },
+  "xion-testnet-2": {
+    "code_ids": {
+      "abstract:account": 95,
+      "abstract:ans-host": 4,
+      "abstract:ibc-client": 8,
+      "abstract:ibc-host": 9,
+      "abstract:module-factory": 6,
+      "abstract:registry": 5,
+      "cw:blob": 3,
+      "polytone:note": 10,
+      "polytone:proxy": 12,
+      "polytone:voice": 11
```

```suggestion
      "cw:blob": 3
```
Oopsie polytone code ids slipped here